polygeist/include/mlir/Dialect/Polygeist/Utils/TransformUtils.h

```diff
@@ -230,6 +231,14 @@ class LoopTools {
                    const LoopVersionCondition &versionCond) const;
 };
 
+//===----------------------------------------------------------------------===//
+// SYCL related functions
+//===----------------------------------------------------------------------===//
+
+/// Return true if the given function is potentially a SYCL kernel body
+/// function.
```

Comment:
Nit: Can we be a bit more specific what "SYCL kernel body function" is here?

Reply:
done.